integrations/testlogger.go

```diff
@@ -98,6 +100,18 @@ func PrintCurrentTest(t testing.TB, skip ...int) func() {
 	}
 	writerCloser.setT(&t)
 	return func() {
+		qs := queue.GetManager().ManagedQueues()
+		for _, q := range qs {
+			if err := q.Flush(10 * time.Second); err != nil {
+				t.Errorf("Flushing queue %s failed with error %v", q.Name, err)
+			}
+		}
+		qs = queue.GetManager().ManagedQueues()
```

Comment:
why is this twice?

Reply:
https://github.com/go-gitea/gitea/pull/10001/files#r371264891 ... hmm ok?

Reply:
Queues can add to Queues...